packages/migrations/src/Command/MigrateCommand.php

```diff
@@ -107,10 +114,14 @@ function ($message) use ($output) {
             }
         );
 
-        $migrationsConfiguration = new Configuration($this->em->getConnection(), $outputWriter);
+        $migrationsConfiguration = new Configuration($this->migrationsConfig, $this->em->getConnection(), $outputWriter);
         $configurationHelper = new ConfigurationHelper($this->em->getConnection(), $migrationsConfiguration);
         $this->getApplication()->getHelperSet()->set($configurationHelper, 'configuration');
 
+        $migrationsConfiguration->setMigrationsDirectory(self::MIGRATIONS_DIRECTORY);
```

Comment:
This actually creates a directory Migrations in the root directory.

Reply:
I will move this back to MigrationsLocator